recipe/meta.yaml

```diff
@@ -10,18 +10,15 @@ source:
   sha256: 3b609b7feba5230e8f6dd8d245ddbefac324c5a6ae4186947670d9ac2cd25573
 
 build:
-  number: 0
-  features:
-    - vc9  # [win and py27]
-    - vc14  # [win and py>=35]
-  skip: true  # [win and py35]
+  number: 1
 
 requirements:
   build:
-    - python         # [win]
-    - toolchain
     - perl 5.22.2.1  # [not win]
     - cmake  # [win]
+    - {{ compiler('fortran') }}
```

Comment:
Hmm...seems to have incorrectly assumed `fortran`. We are not currently supporting that AFAIK.

Reply:
This causes a build failure on Windows.

ref: https://ci.appveyor.com/project/conda-forge/fftw-feedstock/build/1.0.54/job/r4v1pyx3xum259ss